### PR TITLE
MDS and other 2D plots

### PR DESCRIPTION
**Pull request by JasperVanDenBosch**
We currently have an implementation of MDS but not a way to display it yet.

- [x] plot labels
- [x] style axes
- [x] integrate code from #153 
- [x] update import shortcuts
- [x] plot icons
- [x] add to @jooh 's visualization demo?
- [x] plot multiple RDMs

### CODE REVIEW

**Comment by HeikoSchuett** [comment]
take into account #153 here. They did a bit of code for plotting this kind of plot there I think.

**Comment by JasperVanDenBosch** [comment]
> take into account #153 here. They did a bit of code for plotting this kind of plot there I think.

Good point. I've contacted them to ask if it's ok if I combine them

**Comment by JasperVanDenBosch** [comment]
@HeikoSchuett @jooh  I can't seem to get the readthedocs notebook to redraw/update the image. It's currently showing the dots but locally it will show the icons as supposed to. Any idea what I'm doing wrong with the notebook?

The figure is not looking great yet, but the basics are there. Happy to improve if you have any ideas.

I talked to Brandon and he was ok with merging the PRs so to say.

File: rsatoolbox/vis/scatter_plot.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+        rdm_descriptor: (Optional[str]): If provided, this will be used as
+            title for each individual RDM.
+        pattern_descriptor (Optional[str]): If provided, the chosen pattern
+            descriptor will be printed adjacent to each point in the plot
+
+    Returns:
+        Figure: A matplotlib figure in which the plot is drawn
+    """
+    frac, n = math.modf(math.sqrt(rdms.n_rdm))
+    nrows, ncols = math.floor(n), math.floor(n)
```
Comment (jooh) [inline rsatoolbox/vis/scatter_plot.py:37]: Wonder if we should have a consistent utility function that handles this subplot stuff, and allows setting either nrow or ncol, see

https://github.com/rsagroup/rsatoolbox/blob/28aeedc6855046f08bf178ca06b3ee36d647b3aa/rsatoolbox/vis/rdm_plot.py#L111-L120

Comment (jooh) [reply rsatoolbox/vis/scatter_plot.py:37]: (also mplstyle stuff - might help achieve a consistent look and feel across plot code)

https://github.com/rsagroup/rsatoolbox/blob/28aeedc6855046f08bf178ca06b3ee36d647b3aa/rsatoolbox/vis/rdm_plot.py#L37

**Review by jooh — APPROVED** [review]
LGTM overall, but I'm snowed under and haven't had a chance to test drive it myself.

File: rsatoolbox/vis/scatter_plot.py (hunk 1 above)
Comment (JasperVanDenBosch) [reply rsatoolbox/vis/scatter_plot.py:37]: Absolutely, made this issue #207

**Pull request merged**